packages/dashboard/components/consent/pages/index.tsx

```diff
@@ -154,8 +197,16 @@ export default function IndexView() {
       
       return () => clearTimeout(timeout);
     }
-  }, [authLoading, accountsLoading, sessionLoading, loadingMessage]);
+  }, [authLoading, accountsLoading, sessionLoading, isProcessing, loadingMessage]);
+  
+  // ------ CLEANUP ------
 
+  const handleSignOut = async () => {
+    cleanupSession();
```

Comment:
`cleanupSession()` appears to be a duplicate of `clearAuthInfo()` -- we should remove `cleanupSession` and just us `clearAuthInfo()` anywhere `cleanupSession` was previously being referenced